type=class
superclass=RuntimeError
included=
extended=
dynamically_included=
dynamically_extended=
library=pty
aliases=
aliasof=
source_location=refm/api/src/pty.rd:93

子プロセスが終了したり停止した場合に発生する例外です。
